checklists/avd_checklist.en.json

```diff
@@ -518,64 +302,496 @@
             "category": "Identity",
             "subcategory": "Active Directory",
             "text": "Review your organization password expiration policy for accounts used by Azure Files AD integration",
-            "description": "As part of the procedure to integrate Azure File Share and Active Directory authenticaton, an AD account to represent the storage account (file share) will be created. You can choose to register as a computer account or service logon account, see FAQ for details. For computer accounts, there is a default password expiration age set in AD at 30 days. Similarly, the service logon account may have a default password expiration age set on the AD domain or Organizational Unit (OU). For both account types, we recommend you check the password expiration age configured in your AD environment and plan to update the password of your storage account identity of the AD account before the maximum password age. You can consider creating a new AD Organizational Unit (OU) in AD and disabling password expiration policy on computer accounts or service logon accounts accordingly.",
+            "description": "If Azure Files Active Directory (AD) integration is used, as part of the configuration procedure, an AD account to represent the storage account (file share) will be created. You can choose to register as a computer account or service logon account, see FAQ for details. For computer accounts, there is a default password expiration age set in AD at 30 days. Similarly, the service logon account may have a default password expiration age set on the AD domain or Organizational Unit (OU). For both account types, we recommend you check the password expiration age configured in your AD environment and plan to update the password of your storage account identity of the AD account before the maximum password age. You can consider creating a new AD Organizational Unit (OU) in AD and disabling password expiration policy on computer accounts or service logon accounts accordingly.",
             "guid": "2289b3d6-b57c-4fc6-9546-1e1a3e3453a3",
             "severity": "Low",
```

Comment:
Feel like the severity of this should be higher, as this could have a fundamental consequence for file access if the password does expire